### ISSUE TITLE
Support for linux-arm

### ISSUE THREAD

**Issue by gabrielstuff** [SavageCore/node-ffprobe-installer#77]
While testing the project, I encouter : `Unsupported platform/architecture: linux-arm`
Would be nice to have a support for node-ffprobe-installer on linux-arm

**Comment by gabrielstuff** [comment]
Hello, 
I've packaged a version that run on arm (raspberry pi) and I wish to upload it. 
Of course, trying to `npm run upload` failed, as I'm not part of the `@ffprobe-installer` org scope.

This result in the following npm error : 

```
npm notice === Tarball Details ===
npm notice name:          @ffprobe-installer/linux-armhf
npm notice version:       4.2.1
npm notice package size:  16.8 MB
npm notice unpacked size: 28.1 MB
npm notice shasum:        e1ed326127432157b75738d5912210db1979c200
npm notice integrity:     sha512-QF6dZXjdLcUL7[...]LVEUFAjS2fTxA==
npm notice total files:   3
npm notice
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://registry.npmjs.org/@ffprobe-installer%2flinux-armhf - Forbidden
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.
```

What would you suggest @SavageCore in order to contribute ?

**Comment by gabrielstuff** [comment]
Ok tested and it works fine. Would be happy to be able to share it with you.

**Comment by SavageCore** [comment]
Sounds good, thanks for the work. Please send a PR with the new platform and I shall upload to npm and make a release.

**Comment by Netail** [comment]
When will this be released?

**Comment by Netail** [comment]
gabrielstuff's fork for Linux-arm: https://github.com/soixantecircuits/node-ffprobe-installer (If anyone was wondering)

**Comment by JanMalch** [comment]
Any updates on this one?

**Comment by SNRSE** [comment]
Hey are there any updates on this i really need the arm version

**Comment by SavageCore** [comment]
Thanks to SNRSE for https://github.com/SavageCore/node-ffprobe-installer/pull/113. Will add armhf and armel then publish.

**Comment by SavageCore** [comment]
Published v1.1.0 with linux-arm and linux-arm64 support
